components/map/styles/ortho.json

```
@@ -10,6 +10,10 @@
       "tileSize": 256,
       "attribution": "<a target='_blank' href='https://geoservices.ign.fr/documentation/donnees/ortho/bdortho'>© IGN</a>"
     },
+    "openmaptiles": {
+      "type": "vector",
+      "url": "https://openmaptiles.geo.data.gouv.fr/data/france-vector.json"
+    },
```

C'est `decoupage-administratif` dont nous avons besoin. `france-vector` n'est pas adapté car très lourd.
```suggestion
    "decoupage-administratif": {
      "type": "vector",
      "url": "https://openmaptiles.geo.data.gouv.fr/data/decoupage-administratif.json"
    },
```